### ISSUE TITLE
Chapter 11 SELU Accuracy Problem - Keras Default batch size

### ISSUE THREAD

**Issue by ashishthanki** [ageron/handson-ml2#186]
Hi, 

I have been working through the book over the last month, and it has been amazing. Thank you very much. 

While working through the activation functions on chapter 11, specifically, SELU. On the Jupyter Notebook, I realised that the batch size had not been specified yet 55,000 samples were trained - this made sense - until I tried replicating the code, see below. I spent time trying to figure out why my code was only running 1719 samples. Hint: 55,000/32 ~ 1719.

On Chapter 11 Jupyter Notebook  
![image](https://user-images.githubusercontent.com/62998319/84951830-e9d25b80-b0e8-11ea-9b1b-709059b908f0.png)

My Error Code - contained no batch size.
![image](https://user-images.githubusercontent.com/62998319/84953532-9a415f00-b0eb-11ea-929c-c712c2014f39.png)

Keras shows that their default batch size is 32. I am guessing at the time of writing the notebook it was not?
https://keras.rstudio.com/reference/fit.html 

The question I have is, why have they made this change? I understand it speeds up training but the accuracy takes too much of a hit.

Once I set the batch_size=1, the training does take an extremely long time - hours - so it appears to be working. Is that normal? How long does it normally take you to train 55,000 samples? 

Any comment is appreciated =)

**Comment by foxy0** [comment]
Hey! The same question was raised and answered by @Dseal95 in this thread:
https://github.com/ageron/handson-ml2/issues/277#issuecomment-702591890

Basically, fit() method now outputs the number of steps (55,000/32 ~ 1719) as opposed to the total sample size (55,000). In Chapter 10 it is also shown with history.params that the batch_size was indeed 32 though the log showed 55,000.

![Screenshot 2020-12-07 at 20 18 11](https://user-images.githubusercontent.com/63585199/101394935-85d76580-38c9-11eb-9928-06fe0b05db35.png)
